docs/sources/tutorials/get-started.md

```diff
@@ -160,20 +161,24 @@ loki.process "filter_logs" {
   }
 ```
 
-1. `loki.process` is a component that allows you to transform, filter, parse, and enrich log data.
-   Within this component, you can define one or more processing stages to specify how you would like to process log entries before they are stored or forwarded. 
-1. In this example, you create a `loki.process` component named “filter_logs”.
-   This component receives scraped log entries from the `log_scrape` component you created in the previous step. 
-1. There are many ways to [transform, filter, parse, and enrich log data][parse]. In this example, you use the `stage.drop` block to drop log entries based on specified criteria. 
-1. You set the `source` parameter equal to an empty string to denote that scraped logs from the default source, the `log_scrape` component, will be processed.
-1. You set the `expression` parameter equal to the log message that is not relevant to the use case.
-   The log message ".*Connection closed by authenticating user root" was chosen to demonstrate how to use the `stage.drop` block. 
+`loki.process` is a component that allows you to transform, filter, parse, and enrich log data.
+Within this component, you can define one or more processing stages to specify how you would like to process log entries before they're stored or forwarded.
+
+In this example, you create a `loki.process` component named `filter_logs`.
+This component receives scraped log entries from the `log_scrape` component you created in the previous step.
+There are many ways to [transform, filter, parse, and enrich log data][parse].
+In this example, you use the `stage.drop` block to drop log entries based on specified criteria.
```

Comment:
Does is it read strangely to have two paragraphs saying "In this example," with different outcomes for the same example? :thinking: Should they be combined into a single sentence?

Reply:
Mmmm yeah, good point. i'll rework this part.